anndata/_core/anndata.py

```diff
@@ -1897,9 +1898,14 @@ def write_h5ad(
         filename
             Filename of data file. Defaults to backing file.
         compression
-            See the h5py :ref:`dataset_compression`.
+            For [`lzf`, `gzip`], see the h5py :ref:`dataset_compression`.
+            For [`zstd`], see the hdf5plugin :ref:`usage.html <usage>`. Experimental.
```

Comment:
Can simply be this I think

```suggestion
            For [`zstd`], see the hdf5plugin :ref:`hdf5plugin:usage`. Experimental.
```